doc/classes/VisualServer.xml

```diff
@@ -2067,6 +2090,7 @@
 			<argument index="1" name="range_mode" type="int" enum="VisualServer.LightDirectionalShadowDepthRangeMode">
 			</argument>
 			<description>
+				Sets the shadow depth range mode for this directional light. Equivalent to [member DirectionalLight.directional_shadow_depth_range_mode]. See [enum LightDirectionalShadowDepthRangeMode] for options.
```

Comment:
Seems like the property is named `DirectionalLight.directional_shadow_depth_range` (without `_mode`).